packages/common/bucketsConfigJsonObject.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const aws = require('./aws');
+/**
+ * Retrieve the stack's bucket configuration from s3 and return the bucket configuration object.
+ *
+ * @param {string} bucket - system bucket name.
+ * @param {string} stackName - stack name.
+ * @returns {Object} - stack's bucket configuration.
+ */
+async function bucketsConfigJsonObject(
+  bucket = process.env.system_bucket,
+  stackName = process.env.stackName
+) {
+  const bucketsString = await aws.s3().getObject({
```

Comment:
I was going to ask if we should validate the `bucket` and `stackName` arguments, but if they're empty then S3 will probably return a helpful enough error message to make that clear?

Reply:
Yes, but I added a bit of code to make it even clearer.

Reply:
c4c5aed43